### ISSUE TITLE
[4.0] WCAG 1.4.3: Elements must have sufficient color contrast (.form-control-feedback)

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#24261]
**Issue**: `Elements must have sufficient color contrast` ([`color-contrast`](https://dequeuniversity.com/rules/axe/3.1/color-contrast?application=axeAPI))

**Target application**: [Joomla 4 - Administrator Login](http://localhost/joomla-cms/administrator/index.php)

**Element path**: .form-control-feedback

**Snippet**:

    <span class="form-control-feedback">Field value cannot be empty</span>

**How to fix**:

    Fix any of the following:
      Element has insufficient color contrast of 4.27 (foreground color: #c6d2de, background color: #285f9b, font size: 10.1pt, font weight: normal). Expected contrast ratio of 4.5:1

**Environment**:
Chrome version 72.0.3626.121

====

This accessibility issue was found using Accessibility Insights for Web 2.0.5 (axe-core 3.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by hardik-codes** [comment]
Working on it

**Comment by hardik-codes** [comment]
I think it is browser version specific issue 
Because in chrome version 73.0.3683.75 no issues are reported by the tool  http://aka.ms/AccessibilityInsights.
@joomla-cms-bot

**Comment by Harshil99** [comment]
@brianteeman Changing foreground color to #ccd8e2 results to 4.52:1 contrast ratio ? Is it fine or should I make it white?

**Comment by ghost** [comment]
closed as having Pull Request #24265
